### ISSUE TITLE
Tensorflow install should setup up GPU support automatically

### ISSUE THREAD

**Issue by arose13** [tensorflow/tensorflow#21994]
My only real experience with TensorFlow is using it with Keras and sometimes writing some probabilistic objective functions in TensorFlow. Anytime I require GPU support I switch to PyTorch because when I install it, it is automatically set up to run on GPUs. Is there any plan to implement this for TensorFlow?

**Comment by byronyi** [comment]
I do not understand your use cases.

Are there any of them could not be covered by using the `tensorflow-gpu` Python package?

**Comment by coder3101** [comment]
@arose13 There are actually two different packages `tensorflow` and `tensorflow-gpu`. However, Torch doesn't have a different package for GPU.
But I agree it can be/is difficult to set up `tensorflow-gpu` as compared to PyTorch. But here is a thing, Why not get docker Image of `tensorflow-gpu` and use it instead. Here are the [docs](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/dockerfiles) on how to get it.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @drpngx: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @gunan: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @gunan: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by harshini-gadige** [comment]
@arose13  - It looks like you haven't used a template to create this issue. Please resubmit your issue using a template from [here](https://github.com/tensorflow/tensorflow/issues/new/choose). We ask users to use the template because it reduces overall time to resolve a new issue by avoiding extra communication to get to the root of the issue. We will close this issue in lieu of the new one you will create from the template. Thank you for your cooperation.
